blogposts/announcing-sourcegraph-3.8.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Sourcegraph 3.8: Introducing campaign management with Automation and LSIF-based code intelligence"
+author: Christina Forney
+publishDate: 2019-09-01T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.8
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+---
+
+Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
+
+We're excited to announce Sourcegraph 3.8. This release contains lots of behind the scenes work, both foundational and forward looking. Sourcegraph 3.8 introduces faster and more precise [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md)-based code intelligence, and campaign monitoring, the first step towards [code change Automation](https://about.sourcegraph.com/product/automation). In addition to these new concepts, 3.8 introduces search result tabs for better search result discoverability, and the first step towards better monitoring for admins with built-in monitoring tools deployed with every instance.
```

Comment:
First sentence seems unnecessary given the context of the title but also feels like it stutters the third sentence.

```suggestion
This release contains lots of behind the scenes work, both foundational and forward looking. Sourcegraph 3.8 introduces faster and more precise [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md)-based code intelligence, and campaign monitoring, the first step towards [code change Automation](https://about.sourcegraph.com/product/automation). In addition to these new concepts, 3.8 introduces search result tabs for better search result discoverability, and the first step towards better monitoring for admins with built-in monitoring tools deployed with every instance.
```

The 3rd and 4th sentences feel like they should be a list (as it stands they are complex compound sentences), but we already kind of have a list below, so I am wondering if there is a way to simplify this.